pkg/kubelet/stats/provider.go

```diff
@@ -203,6 +204,9 @@ func (p *Provider) GetRawContainerInfo(containerName string, req *cadvisorapiv1.
 }
 
 // HasDedicatedImageFs returns true if a dedicated image filesystem exists for storing images.
+// KEP Issue Number 4191: Enhance this to allow for container or image between separated.
```

Comment:
We want to clarify that we support the case of writeable layer locationed with rest of the root fs and read only layers of images on a separate disk.

Reply:
updated.